src/components/App/App.tsx

```diff
@@ -137,7 +161,7 @@ const App = (props) => {
     return () => {
       window.removeEventListener('scroll', handleScroll);
     };
-  });
+  }, []);
```

Comment:
doesn't this change the behavior to only happen once?

Reply:
isn't it the expected behavior? clicking on the "back to top" button takes the user to the top of the screen and then the button disappears. user won't be able to click it multiple times.

Reply:
If they scroll back down though we want to reenable it right?